app/slashcommands/command_share.go

```diff
@@ -240,9 +240,19 @@ func (sp *ShareProvider) doInviteRemote(a *app.App, args *model.CommandArgs, mar
 		return responsef("Must specify a valid remote cluster id to invite.")
 	}
 
-	rc, err := a.GetRemoteCluster(remoteId)
+	// Check if channel is shared or not.
+	has, err := a.HasSharedChannel(args.ChannelId)
 	if err != nil {
-		return responsef("Remote cluster id is invalid: %v", err)
+		return responsef("Error while checking if shared channel exists: %v", err)
+	}
+	if !has {
+		// If it doesn't exist, then create it.
+		sp.doShareChannel(a, args, margs)
```

Comment:
`doShareChannel` returns a string that becomes an ephemeral message notifying the user the channel has been shared, or any error.  Can we add that to the response returned below?  Just prepending it to the response below with a newline separator should be enough.